### PR TITLE
Feature 311 ush rearrange

### PR DESCRIPTION
**Pull request by georgemccabe**

### CODE REVIEW

File: metplus/wrappers/cyclone_plotter_wrapper.py [hunk 1]
```diff
@@ -22,11 +22,10 @@
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
 from cartopy.mpl.gridliner import LONGITUDE_FORMATTER, LATITUDE_FORMATTER
-import met_util as util
 import produtil.setup
-import config_metplus
-from command_builder import CommandBuilder
 
+from ..util import met_util as util
+from . import CommandBuilder
```
Comment (hankenstein2) [inline metplus/wrappers/cyclone_plotter_wrapper.py:28]: Do you not need a config_metplus here or was it forgotten?

**Review by hankenstein2 — APPROVED** [review]
Looks good to me

File: metplus/wrappers/cyclone_plotter_wrapper.py (hunk 1 above)
Comment (georgemccabe) [reply metplus/wrappers/cyclone_plotter_wrapper.py:28]: 'import config_metplus' was in there before, but I searched and it wasn't used anywhere in the code.

**Comment by fentoad72** [comment]
Closing.  This work has been completed and reviewed

**Pull request merged**